packages/components/pie-tag/src/tag.scss

```diff
@@ -162,7 +162,7 @@
     &.c-tag--brand-06 {
         --tag-bg-color: var(--dt-color-support-brand-06);
         --tag-color: var(--dt-color-content-light);
-        @include tag-interactive-states('--dt-color-support-brand-06');
```

Comment:
for interactive mode should there be a pointer cursor?

Reply:
good catch, I also noticed that with TingTing earlier and I'm going to to refine a bug ticket for it tomorrow. we should also disable user-select for isInteractive 🤾

Reply:
🤝